### DISCUSSION TITLE
Nulleinspeisung Zendure Hub 2000 + AB2000 + HMS-1600-4T + ShellyPro3EM + OpenDTUoB

### DISCUSSION THREAD

**Discussion by Josef1966** [hoylabs/OpenDTU-OnBattery#1379] [Q&A]
Hallo, ich betreibe einen Hub2000 + AB2000 + HMS-1600-4T + Shelly Pro 3EM entsprechend der anhängenden Darstellung mit dynamischer Regelung über OpenDTU onBattery am Wechselrichter. Die Einstellungen in der DTU habe ich angehängt.

In der Zendure App sind alle Energiepläne deaktiviert und der Ausgang am Hub ist auf 1200 Watt gestellt.

Leider funktioniert die Nulleinspeisung so nicht. Der Wechselrichter speist wesentlich mehr ins Netz als eingestellt. Dabei wird sogar Energie aus dem Akku gezogen und ins öffentliche Netz eingespeist. Screenshots von OpenDTU und Zendure sind im Anhang.

Was ist da verkehrt? Welche Einstellungen bzw. Parameter müssen anderes konfiguriert werden, damit es funktioniert?

![Speicher_Hoymiles_Zendure_WR-2-Eingänge](https://github.com/user-attachments/assets/7e00e917-a19c-4e54-ba95-1bb540e1e71b)
![OpenDTU-oB_DPL](https://github.com/user-attachments/assets/f36678b1-85d7-47fe-b0d0-a847c5f4e2d2)
![OpenDTU-oB_Nulleinspeisung](https://github.com/user-attachments/assets/bda342cc-7de9-4a33-95e2-b7254cb3fda8)
![Zendure_Nulleinspeisung](https://github.com/user-attachments/assets/b3215cb6-584d-47ed-8be4-c64b3a7d3d75)

**Comment by spcqike** [comment]
Der hub hat nur einen Ausgang? 
dann solltest du die Stecker und Kabel überprüfen. Der hoymiles erzeugt auf einem Eingang wesentlich mehr, als auf dem anderen. Was so eigentlich nicht geht. Das kommt vor, wenn die Leitungen von der Batterie zum wechselrichter ungleichmäßig sind.

**Reply by Josef1966** [reply]
Ja der Hub hat nur einen Ausgang. Vielleicht sollte ich mal andere 2 Eingänge am WR testen?

**Reply by spcqike** [reply]
Versuch es. 

Oder andere Kabel. Andere Y Kabel. 

Was auch immer. Am Ende muss jeder Eingang ungefähr das gleiche erzeugen. 

Eventuell auch mal das überskalieren deaktivieren. Testweise.
